.github/workflows/integration_test.yaml

```diff
@@ -10,11 +10,10 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
-      extra-arguments: >
-        -m "not (requires_secret)"
-        --openstack-rc ${GITHUB_WORKSPACE}/openrc
-        --kube-config ${GITHUB_WORKSPACE}/kube-config
-        --screenshot-dir /tmp
+      extra-arguments: >-
+        --openstack-rc=${GITHUB_WORKSPACE}/openrc
+        --kube-config=${GITHUB_WORKSPACE}/kube-config
+        --screenshot-dir=/tmp
       modules: '["test_addon", "test_core", "test_ingress", "test_cos_grafana", "test_cos_loki", "test_cos_prometheus"]'
```

Comment:
Would you please the `test_external` module here? The `test_external` module contains all the tests that require secrets.